src/components/IconGrid.tsx

```diff
@@ -1,221 +1,41 @@
-                className={classnames(styles.styleToggle, {
-                  [styles.styleToggleSelected]: searchStyleValue === 'outline'
-                })}
-                onClick={() => {
-                  dispatch(setStyle('outline'));
-                }}
-              >
-                Outline
-              </button>
-            </div>
-          </div>
-        </div>
-
-        {categories.map((c, categoryIndex) => (
-          <div key={categoryIndex}>
-            {(!isFiltered ||
-              c.icons.some((i) => {
-                return iconsToRender.includes(i);
-              })) && <CategoryHeading>{c.title}</CategoryHeading>}
-            <div className={styles.iconGrid}>
-              {c.icons.map((i, iconIndex) => (
-                <IconTile
-                  key={iconIndex}
-                  icon={i}
-                  iconStyle={searchStyleValue}
-                  visible={!isFiltered || iconsToRender.includes(i)}
-                  onClick={(iconType: string) => {
-                    // uses the "as" property instead of the "url" to keep the route from changing which causes all of the icons to re-render each time. See the useEffect() above that captures the back/forward buttons to handle the URL changes
-                    router.push(
-                      '/',
-                      `/icon/${iconType}/${i.category}/${i.id}`,
-                      {
-                        shallow: true,
-                        scroll: false
-                      }
-                    );
-
-                    setModalIcon({ icon: i, iconType });
-                  }}
-                />
-              ))}
-            </div>
-          </div>
-        ))}
-        {modalIcon && (
-          <IconTileModal
-            icon={modalIcon.icon}
-            iconType={modalIcon.iconType}
-            isOpen={modalIcon !== undefined}
-            onClose={() => {
-              router.push('/', undefined, {
-                shallow: true,
-                scroll: false
-              });
-
-              setModalIcon(undefined);
-            }}
-          />
-        )}
-      </main>
-      <footer>
-        All icons are open source, licensed under a{' '}
-        <a href="/about#license" className={styles.link}>
-          Creative Commons CC0 License
-        </a>
-        . This website is also open source, under an{' '}
-        <a href="/about#license" className={styles.link}>
-          MIT license
-        </a>
-        .
-      </footer>
+    <div className={styles.iconGrid}>
+      {icons.map((icon, iconIndex) => (
+        <IconTile
+          key={iconIndex}
+          icon={icon}
+          iconStyle={style}
+          visible={true}
+          onClick={(iconType: string) => {
+            // uses the "as" property instead of the "url" to keep the route from changing which causes all of the icons to re-render each time. See the useEffect() above that captures the back/forward buttons to handle the URL changes
```

Comment:
this references the `useEffect()` that's now in `LandingPage`

Reply:
```suggestion
            // uses the "as" property instead of the "url" to keep the route from changing which causes all of the icons to re-render each time. See the useEffect() in LandingPage that captures the back/forward buttons to handle the URL changes
```